jenkins/debian/debos/scripts/stretchltp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Important: This script is run under QEMU
+
+set -e
+
+# Build-depends needed to build the test suites, they'll be removed later
+BUILD_DEPS="
+            acl \
+            attr \
+            autoconf \
+            automake \
+            gawk \
+            git \
+            gcc \
+            libaio1 \
+            libc6-dev \
+            libcap2 \
+            make \
+            openssl \
+            zip \
+"
+
+apt-get install --no-install-recommends -y  ${BUILD_DEPS}
+
+BUILDFILE=/build_info.txt
+echo '  "tests_suites": [' >> $BUILDFILE
+
+# Build and install IO performance suites
+########################################################################
+TMP_TEST_DIR="/tmp/tests"
+mkdir -p ${TMP_TEST_DIR} && cd ${TMP_TEST_DIR}
+
+TESTDIR=ltp
+TEST_REPO=https://github.com/linux-test-project/ltp.git
+
+git -c http.sslVerify=false clone --depth=1 ${TEST_REPO}
+cd ${TESTDIR}
+
+echo '    {"name": "${TESTDIR}", "git_url": "${TEST_REPO}", "git_commit": ' \"`git rev-parse HEAD`\" '}' >> $BUILDFILE
+
+make autotools
+./configure --prefix=/opt/ltp
```

Comment:
Why not `--prefix=/tmp/tests/ltp` like for the other test suites? (see `stretch-igt.sh` and `stretch-v4l2.sh`)

Reply:
Why install it into /tmp/tests/, that wont be in the image. Why not just use the default for the project ? and that is /opt/ltp, since LTP has its own directory hierarchy. I don't see the point in installing it into /tmp/tests/ltp and then copy it over to another location like stretch-igt.sh and stretch-v4l2.sh does it.
However, they are built and copied over since you run strip on them right ?
Can that be done by passing flag '-s' to gcc, if yes then stretch-(igt|v4l2).sh can have the configure's --prefix flag point to /usr or maybe I miss something ?

Reply:
Agreed that test suites could be installed in `/opt`.